critiquebrainz/frontend/external/musicbrainz_db/helpers.py

```diff
@@ -22,30 +20,35 @@ def entity_relation_helper(db, target_type, source_type, source_entity_id):
         options(joinedload("link", innerjoin=True)).\
         options(joinedload("link.link_type", innerjoin=True))
     if relation.entity0.property.mapper.class_ == relation.entity1.property.mapper.class_:
-        links = get_relationship_links(relation, query, "entity0", "entity1", source_entity_id)
-        links.extend(get_relationship_links(relation, query, "entity1", "entity0", source_entity_id))
+        _relationship_link_helper(relation, query, "entity0", "entity1", target_type, source_entity_ids, includes_data)
+        _relationship_link_helper(relation, query, "entity1", "entity0", target_type, source_entity_ids, includes_data)
     else:
         if source_model == relation.entity0.property.mapper.class_:
-            links = get_relationship_links(relation, query, "entity0", "entity1", source_entity_id)
+            _relationship_link_helper(relation, query, "entity0", "entity1", target_type, source_entity_ids, includes_data)
         else:
-            links = get_relationship_links(relation, query, "entity1", "entity0", source_entity_id)
-    return links
+            _relationship_link_helper(relation, query, "entity1", "entity0", target_type, source_entity_ids, includes_data)
 
 
-def get_relationship_links(relation, query, source_attr, target_attr, source_entity_id):
+def _relationship_link_helper(relation, query, source_attr, target_attr, target_type, source_entity_ids, includes_data):
     """Get relationship links between two entities.
 
     Args:
         relation (mbdata.model): Model relating the two entities.
         query (Session.query): Query object.
         source_attr (str): 'entity0' or 'entity1' based on which represents source model in relation table.
         target_attr (str): 'entity0' or 'entity1' based on which represents target model in relation table.
-        source_entity_id (int): ID of the source entity.
+        target_type (str): Type of the target entity.
+        source_entity_ids (list): IDs of the source entity.
+        includes_data (dict): Dictionary containing the includes data of entities.
 
-    Returns:
-        List of objects of relationships between entities.
+    Note:
```

Comment:
This is not really a standard way of writing an extended description. Check https://github.com/metabrainz/guidelines/blob/master/Python.md#docstrings. Here's how you could write this:
```python
    """Get relationship links between two entities.

    Keep in mind that includes_data (dict) is altered to contain the relationship objects
    keyed by the source entity MBIDs.

    Args:
        relation (mbdata.model): Model relating the two entities.
        query (Session.query): Query object.
        source_attr (str): 'entity0' or 'entity1' based on which represents source model in relation table.
        target_attr (str): 'entity0' or 'entity1' based on which represents target model in relation table.
        target_type (str): Type of the target entity.
        source_entity_ids (list): IDs of the source entity.
        includes_data (dict): Dictionary containing the includes data of entities.
    """
```